azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -278,6 +283,11 @@ private void processQueuedFlows() throws ExecutorManagerException {
         rateLimiter.acquire();
         logger.info("Starting dispatch for {} execution.", executionId);
         Runnable worker = new ExecutionDispatcher(executionId);
+        // Fetch an executable flow based on execution id and report an dispatching event
+        ContainerizedDispatchManager.this.fireEventListeners(Event.create(fetchFlow(executionId),
```

Comment:
You should use fetchFlow only once. By that I mean create ExecutableFlow variable.

Reply:
Made changes as requested.